src/test/java/org/openmrs/contrib/qaframework/automation/EditDemographicsAndContactInfoSteps.java

```diff
@@ -94,12 +94,12 @@ public void systemSavesUpdatedDemographics() {
 	@When("a user clicks on Edit link under contact info section from Registration Summary Page")
 	public void loadContactInfoEditSection() {
 		registrationSummaryPage.clickOnShowContact();
-		registrationEditSectionPage = (RegistrationEditSectionPage) registrationSummaryPage.clickOnEditContact().waitForPage();
+		registrationEditSectionPage = (RegistrationEditSectionPage) registrationSummaryPage.clickOnEditContact();
 	}
 
 	@Then("the system loads the edit contact information section")
 	public void systemLoadsEditContactInfoSection() {
-		assertPage(registrationEditSectionPage.waitForPage());
+		assertPage(registrationEditSectionPage);
```

Comment:
Refer to the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/153/commits/7066181db4f42a7fd017c1d687fee0fa27e125ac#r749389133)